src/components/global/localNav/localNavLink.ts

```diff
@@ -85,51 +93,39 @@ export class LocalNavLink extends LitElement {
   @state()
   _leaveTimer: any;
 
-  /** Menu positioning
-   * @internal
-   */
-  @state()
-  menuPosition: any = {};
-
   override render() {
     const classes = {
       'level--1': this._level == 1,
       'level--2': this._level == 2,
       'level--3': this._level == 3,
+      'sub-level': this._level > 1,
```

Comment:
comparing to the figma, there isn't a fourth level with the filled in bullet to review in the storybook. assuming that's taken care of, or maybe it was deemed unnecessarily in the nesting?

Reply:
Cleaned this up a bit. The updates kinda made nesting recursive/infinite, although the top level still has some unique considerations with the collapsed nav state.